### ISSUE TITLE
SUPPORTS_EVENT_OPTIONS is never true (even on chrome)

### ISSUE THREAD

**Issue by rwjblue** [buschtoens/ember-on-modifier#41]
Noticed this while working on upstreaming things into Ember.

The bug is here:

https://github.com/buschtoens/ember-on-modifier/blob/a96340f86a59811a4b7bc12394a69a6f410bb72d/addon/utils/event-listener.js#L22

It should be:

```js
    div.addEventListener('click', () => counter++, { once: true );
```

Otherwise the `counter` at the end will always be `2`...

**Comment by buschtoens** [comment]
🤦‍♂️

Thanks! I'll make sure to cover this with tests now.

**Comment by buschtoens** [comment]
Fixed in [`v0.12.1`](https://github.com/buschtoens/ember-on-modifier/releases/tag/v0.12.1).
